internal/services/cosmos/cosmosdb_account_resource.go

```diff
@@ -110,6 +110,53 @@ func resourceCosmosDbAccount() *pluginsdk.Resource {
 				}, true),
 			},
 
+			"analytical_storage_configuration": {
```

Comment:
we can probably make this
```suggestion
			"analytical_storage": {
```

Reply:
Updated